docker/README.md

```diff
@@ -1 +1,7 @@
-[TODO: fill in docs for docker examples and how to run]
\ No newline at end of file
+## SparseML docker image
+This directory contains the Dockerfile to create minimal SparseML docker image.
+This image is based off the official NVIDIA development Ubuntu 18.04.5 LTS image (with CUDA support).
+
+In order to build and launch this image, run from the root directory:
+- for compute platform CUDA 10.2: `docker build --build-arg CUDA_VERSION=10.2 -t sparseml_docker . && docker run -it --gpus all sparseml_docker`
+- for compute platform CUDA 11.1: `docker build --build-arg CUDA_VERSION=11.1 -t sparseml_docker . && docker run -it --gpus all sparseml_docker`
```

Comment:
let's break down this example usage section into `build` which is separate for either cuda version and then `run` which will be the same and we can provide example commands to run / how to get into an interactive shell

Reply:
Ok, the import problems are fixed now.

But the nvidia images are still faulty, so I am keeping the hack that I mentioned inplace.